backend/database/README.md

```diff
@@ -18,9 +18,14 @@
 6. [Test your migration](#test-a-migration)
 7. Check that [orm.py](../layers/persistence/orm.py) matches up with your changes.
 8. Once you've completed the changes, create a PR to get the functions reviewed.
-9. Once the PR is merged, migrations will be run as part of the deployment process to each env.
-10. [Connect to Remote RDS](#connect-to-remote-rds) to single-cell-dev
-11. In a new terminal, complete the migration in the single-cell-dev test env by running:
```

Comment:
The last two instructions -- connect to remote, and manually run migration -- are no longer necessary, right @Bento007 ? They weren't for my most recent migration.

Reply:
they are still valid if we need to do it manually.